packages/libs/error-stack/.gitattributes

```
@@ -0,0 +1,2 @@
+tests/snapshots/*.snap linguist-generated=true
```

This should also cover `snapshots/doc/*.snap`
```suggestion
tests/snapshots/**.snap linguist-generated=true
```

---

Now it's working 🎉